requirements/base.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    make upgrade
 #
-pbr==5.4.4                # via stevedore
+pbr==5.4.5                # via stevedore
```

Comment:
I've filed a ticket with pbr for their changelog having disappeared: https://bugs.launchpad.net/pbr/+bug/1871841

In the meantime, this is the only change: https://opendev.org/openstack/pbr/commit/b821df5f232e0843aced66b508c1d7355887bc9b